### DISCUSSION TITLE
Should we remove the "abandon all my bases" option?

### DISCUSSION THREAD

**Discussion by andy5995** [netpanzer/netpanzer#171] [General]
Should we remove the "abandon all my bases" option? Does anyone know of a point to that option?

I don't remember seeing that option. There's one that will turn production off for all bases. I assume that is so enemy tanks can't kill newly produced units when there's not enough friendly tanks to defend.

**Comment by winrid** [comment]
I recall "abandon" and "abandon all" - it's kinda hard to see and not clear when it does when you click it.
